lib/hushed/blackboard.rb

```diff
@@ -40,5 +42,40 @@ def build_document(type, contents)
       end
       namespace.const_get(type).new(:io => contents) if namespace
     end
+
+    def retrieve_latest(document_name)
+      blackboard = client.from_quiet_bucket
+      documents = inventory_summary_documents(blackboard, document_name)
+      get_latest_summary_report(documents)
```

Comment:
following the "Ask don't tell" principle and with the renamed methods, I'd do something like this (no need to give the blackboard as an argument).
```
def retrieve_latest(document_name_prefix)
    documents = documents_by_date(document_name_prefix)
    get_latest_created_document(documents)
end
```
Looks cleaner to me